lib/vty.c

```diff
@@ -3782,9 +3783,6 @@ static int vty_mgmt_get_tree_result_notified(
 	size_t len, int partial_error)
 {
 	struct vty *vty;
-	struct lyd_node *dnode;
```

Comment:
As discussed in slack, these changes need to be reverted so that we support using LYB up to right before converting to send to the user.